src/logic/operate.js

```diff
@@ -0,0 +1,39 @@
+import Big from 'big.js';
+
+const operates = (numberOne, numberTwo, operation) => {
+  const firstInput = Big(+numberOne);
+  const secondInput = Big(+numberTwo);
+  let result;
+
+  switch (operation) {
+    case '+':
+      result = firstInput.plus(secondInput);
+      break;
+
+    case '-':
+      result = firstInput.minus(secondInput);
+      break;
+
+    case 'x':
+      result = firstInput.times(secondInput);
+      break;
+
+    case '÷':
+      result = firstInput.div(secondInput);
+      break;
```

Comment:
- [ ] Please handle error from division by zero here as it should fatally crash the application if not gracefully handled.